modules/personalize_blocks/personalize_blocks.module

```diff
@@ -4,19 +4,6 @@
  * Implements hook_menu().
  */
 function personalize_blocks_menu() {
-  $items['admin/structure/personalize/variations/personalize-blocks'] = array(
-    'title' => 'Blocks',
-    'description' => 'Create personalized blocks.',
-    'page callback' => 'personalize_blocks_list',
-    'access callback' => 'user_access',
-    'access arguments' => array('manage personalized content'),
-    'type' => MENU_LOCAL_TASK,
-    'file' => 'personalize_blocks.admin.inc',
-  );
-  $items['admin/structure/personalize/variations/personalize-blocks/list'] = array(
-    'title' => 'List personalize blocks',
-    'type' => MENU_DEFAULT_LOCAL_TASK,
-  );
   $items['admin/structure/personalize/variations/personalize-blocks/add'] = array(
     'title' => 'Add personalized block',
```

Comment:
I think that the add item here is getting orphaned because it is defined as a tab within the full listing.  It's menu type should be updated.